satpy/readers/viirs_compact.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-# Copyright (c) 2014, 2015 Martin Raspaud
+# Copyright (c) 2014 - 2019 PyTroll
```

Comment:
I think we usually do `2014-2019` (no spaces) and it should be `Satpy developers`.